Update/omot_010_togoodend.txt

```diff
@@ -320,25 +320,25 @@ void main()
 //rその際、入江が『東京』に向けて作り上げたレポートの一部を秘かに盗み見る機会があったのだが、そこに書かれてあった『オヤシロさま』という文言――。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "その際、入江が『東京』に向けて作り上げたレポートの一部を秘かに盗み見る機会があったのだが、そこに書かれてあった『オヤシロさま』という文言——。",
-			NULL, "At the time, I managed to steal part of a report Irie made for \"Tokyo\" and found the name \"Oyashiro-sama\" written there...", Line_Normal);
+			NULL, "At the time, I accidentally snuck a read part of a report Irie made for \"Tokyo\" and found the name \"Oyashiro-sama\" written there...", Line_Normal);
 	ClearMessage();
 
 //rレナは、自分にまつわる異常状態を全て、幼い時に聞いた『オヤシロさまの崇り』によるものだと解釈して、その原因を自らの家族愛の至らなさに求めたのだという。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "レナは、自分にまつわる異常状態を全て、幼い時に聞いた『オヤシロさまの崇り』によるものだと解釈して、その原因を自らの家族愛の至らなさに求めたのだという。",
-			NULL, "Rena interprets all her abnormal circumstances to a concept she heard when she was young called \"the curse of Oyashiro-sama\" as a way of alleviating her lack of familial love.", Line_Normal);
+			NULL, "It said that Rena interpreted all her abnormal circumstances to a concept she heard when she was young called \"the curse of Oyashiro-sama\" resulted from her lack of familial love.", Line_Normal);
 	ClearMessage();
 
 //r…これは、安易に責任転嫁する年相応の子供では決して起こりえないことで、それだけレナが心優しく、家族を愛し…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…これは、安易に責任転嫁する年相応の子供では決して起こりえないことで、それだけレナが心優しく、家族を愛し…。",
-			NULL, "...It's normal for a child of her age to settle on something or someone to push responsibility on to, but Rena really does love her family...", Line_Normal);
+			NULL, "...It's normal for a child of her age to settle on something or someone to push responsibility on to, but Rena didn't do such a thing because paradoxically she really did love her family...", Line_Normal);
```

Comment:
```suggestion
			NULL, "...It's normal for a child of her age to settle on something or someone to push responsibility on to, but Rena didn't because she really did love her family...", Line_Normal);
```